src/Jarvis.Core/Interop/Kernel32.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Jarvis.Core.Interop
+{
+    public static class Kernel32
```

Comment:
Would prefer to put this in `public static partial class Win32` and name the file `Win32.Process` or similar.